examples/webgpu_postprocessing_3dlut.html

```diff
@@ -158,8 +158,7 @@
 				controls.target.set( 0, 0, - 0.2 );
 				controls.update();
 
-				gui = new GUI();
-				gui.width = 350;
```

Comment:
Since the line had no effect so far, I would simply remove it without adding it to the ctor. The GUI seems fine as it is.